doc/source/reference/maskedarray.baseclass.rst

```diff
@@ -82,13 +85,17 @@ Attributes and properties of masked arrays
    The type of the data can be accessed through the :attr:`baseclass`
    attribute.
 
+.. _ma-mask:
+
 .. attribute:: MaskedArray.mask
```

Comment:
What happens if we change these to `autoattribute`?

Reply:
nice, I moved the attribute documenation to docstrings and used autoattribute, which cleaned up the need for labels and also extended the help(ma.<attribute>) documentation